### PR TITLE
OCPERT-5 add 4.16 auto release jobs

### PR DESCRIPTION
**Pull request by tomasdavidorg**
https://issues.redhat.com/browse/OCPERT-5

### CODE REVIEW

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__automated-release-stable-4.16-upgrade-from-stable-4.16.yaml [hunk 1]
```diff
@@ -57,6 +57,16 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
+- as: aws-ipi-f999
```
Comment (tomasdavidorg) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__automated-release-stable-4.16-upgrade-from-stable-4.16.yaml:60]: @rioliu-rh should we add these jobs or update the existing ones?

Comment (rioliu-rh) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__automated-release-stable-4.16-upgrade-from-stable-4.16.yaml:60]: I suggest to do clean up for old jobs, and add new jobs with typical profiles

**Comment by rioliu-rh** [comment]
test jira link [issues.redhat.com/browse/OCPERT-5](https://issues.redhat.com/browse/OCPERT-5)

**Comment by rioliu-rh** [comment]
@tomasdavidorg let's unhold this PR, thanks

**Comment by rioliu-rh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-automated-release-aws-ipi-f999

**Comment by rioliu-rh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-automated-release-stable-4.16-upgrade-from-stable-4.15-gcp-ipi-f99

**Comment by rioliu-rh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-automated-release-stable-4.16-upgrade-from-stable-4.15-gcp-ipi-f999

**Comment by rioliu-rh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-automated-release-stable-4.16-upgrade-from-stable-4.16-gcp-ipi-f999

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__automated-release-stable-4.16-upgrade-from-stable-4.15.yaml [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    namespace: ocp-kni
+    tag: dev-scripts
+  openstack-installer:
+    name: "4.15"
+    namespace: ocp
+    tag: openstack-installer
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.15"
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__automated-release-stable-4.16-upgrade-from-stable-4.15.yaml:21]: I think `tests-private-postupg` should use 4.16 here.

Comment (rioliu-rh) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__automated-release-stable-4.16-upgrade-from-stable-4.15.yaml:21]: +1

Comment (rioliu-rh) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__automated-release-stable-4.16-upgrade-from-stable-4.15.yaml:21]: even we don't run pre/post upgrad test in upgrade job. anyway, we need to correct this config setting

Comment (rioliu-rh) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__automated-release-stable-4.16-upgrade-from-stable-4.15.yaml:21]: @tomasdavidorg let us apply this change, then I'll add label lgtm for this PR.

**Comment by rioliu-rh** [comment]
@Prucek Can you help to override this failed test again, thanks

**Pull request merged**